src/mbgl/layout/pattern_layout.hpp

```diff
@@ -13,7 +13,7 @@ class PatternDependency {
     std::string max;
 };
 
-using PatternLayerMap = std::unordered_map<std::string, PatternDependency>;
+using PatternLayerMap = std::map<std::string, PatternDependency>;
```

Comment:
Any specific reason for this change?

Reply:
https://groups.google.com/a/chromium.org/forum/#!topic/chromium-dev/rdxOHKzQmRY
This map does not seem to be big enough